myuw/static/js/card/schedule/visual.js

```diff
@@ -90,8 +94,20 @@ var VisualScheduleCard = {
         var set_meeting = function(course_data) {
             $.each(course_data.sections, function(section_index) {
                 var section = this;
+                section.has_eos_dates = false;
                 $.each(section.meetings, function(){
                     var meeting = this;
+
+                    if(meeting.eos_start_date) {
+                        if(!section.has_eos_dates) {
+                            section.has_eos_dates = true;
+                        }
+                    }
```

Comment:
Could this be changed to

```
section.has_eos_dates = meeting.eos_start_date && !section.has_eos_date
```

?

Reply:
This may set section.has_eos_dates from true to false.